src/main/java/org/dcsa/ovs/model/transferobjects/OperationsEventTO.java

```diff
@@ -2,12 +2,12 @@
 
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import org.dcsa.core.events.model.OperationsEvent;
 import org.dcsa.core.events.model.transferobjects.PartyTO;
-import org.dcsa.ovs.model.base.AbstractOperationsEvent;
 
 @Data
 @EqualsAndHashCode(callSuper = true)
-public class OperationsEventTO extends AbstractOperationsEvent {
+public class OperationsEventTO extends OperationsEvent {
```

Comment:
Shouldn't this class also be part of EventCore?

Reply:
Has been moved to EventCore